src/components/SearchResults.jsx

```diff
@@ -0,0 +1,36 @@
+import { React, useState } from "react";
```

Comment:
You dont need to import state here as it isnt being used ion the component

Reply:
probably left over from a previous version of the code where state was used.